app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+
+import { Button, Alert, Toastr } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import Menu from "./Menu";
+import Table from "./Table";
+
+const Contacts = () => {
+  const [showMenu, setShowMenu] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+
+  const handleSubmit = () => {
+    setShowDeleteAlert(showDeleteAlert => !showDeleteAlert);
+    Toastr.success("Contact deleted successfully.");
+  };
+  return (
+    <>
+      <Menu showMenu={showMenu} />
+      <Container>
+        <Header
+          title="All Contacts"
+          actionBlock={
+            <Button
+              icon="ri-add-line"
+              label="Add New Contact"
+              onClick={() => {}}
+            />
+          }
+          menuBarToggle={() => {
+            setShowMenu(prevSetShowMenu => !prevSetShowMenu);
+          }}
+          searchProps={{
+            value: searchTerm,
+            onChange: e => setSearchTerm(e.target.value),
+          }}
+        />
+        <Table
+          showAlert={() =>
+            setShowDeleteAlert(showDeleteAlert => !showDeleteAlert)
+          }
+        />
+        <Alert
+          closeButton={false}
+          isOpen={showDeleteAlert}
+          message="Are you sure you want to delete this contact? These changes cannot be undone."
```

Comment:
Why have you mentioned `These changes`? There is only one change that's happening here. That is deleting a single contact. Fix the grammar in the message.

Reply:
Fixed

Reply:
Why is the `C` in `Change` capitialized? Also, I have been seeing a lot of grammatical mistakes in your English sentences. Moving forward, make sure to use either Grammarly or Google Docs or both combined or some other tool to make sure that your grammar is on point before writing any English sentence in your code or when you reply.

https://handbook.neetokb.com/articles/english-grammar

Reply:
Ok will take of same moving forward
I will check the sentence from google docs enabled Grammarly.